### ISSUE TITLE
Support for managing Debian-based OS

### ISSUE THREAD

**Issue by BrianSipos** [jednoprsak/PuppetNetworkManagerModule#3]
The current `os.pp` limits this module to not being able to manage Ubuntu or other Debian-based OS. It would be helpful to add this ability, because the module is very close to working on Ubuntu otherwise. If helpful I could submit a small patch, after some testing, for Ubuntu.

**Comment by jednoprsak** [comment]
Yes, please. If you have condition to test it propperly for debian, do it :-) . Please submit your small patch

**Comment by jednoprsak** [comment]
Samuraiii added a Debian support for the module. Please could you test a new release beta 1.0.0-rc1 of the module?
